xbmc/network/mdns/ZeroconfBrowserMDNS.cpp

```diff
@@ -30,8 +30,8 @@ using namespace std::chrono_literals;
 extern HWND g_hWnd;
 
 CZeroconfBrowserMDNS::CZeroconfBrowserMDNS()
+: m_browser()
```

Comment:
=> m_browser(NULL) or m_browser(nullptr), to make really clear what the intention is.